src/main/java/com/ericsson/ei/exception/ReplacementMarkerException.java

```diff
@@ -0,0 +1,10 @@
+package com.ericsson.ei.exception;
+
+public class ReplacementMarkerException extends RuntimeException {
```

Comment:
This should maybe not be a RuntimeException.

Reply:
It is an Exception now.